### PR TITLE
added views

### PR DESCRIPTION
**Pull request by AtillaTahak**
Feature: Views
In this update:

Impelemented the given design.
Created seed.rb file for adding data to the database as described.
Have a nice CR 👍

### CODE REVIEW

File: app/views/users/show.html.erb [hunk 1]
```diff
@@ -6,9 +6,10 @@
...
       <div class="col-md-8">
         <div class="card-body">
-          <h5 class="card-title"> User 1</h5>
-          <p class="card-text">This Area For User knowledge </p>
-          <p class="card-text"><small class="text-muted">Number of posts: x</small></p>
+          <h5 class="card-title"> <%= @post.title %> by <%= @user.name %></h5>
+          <p class="card-text"><small class="text-muted">Comment(s): <%= @post.comments_counter%></small></p>
+          <p class="card-text"><small class="text-muted">Likes(s): <%= @post.likes_counter%></small></p>
+
         </div>
```
Comment (rloterh) [inline app/views/users/show.html.erb]: - [ ] Contents for _show_ action are being manually generated by your app. Kindly ensure that the details of each post are rendered according to the corresponding post by the current user. You can improve upon this by making your code to be similar to:
```ruby
# app/views/users/show.html.erb

[...]

<% @user.recent_post.each do |post| %>

<h4><%= link_to post.title, user_post_path(post) %></h4>
<p><%= post.text %></p>
<p>
<small>Comments: <%= post.comments_counter || 0 %>,</small> 
<small>Likes: <%= post.likes_counter || 0 %></small>
</p>

<% end %>

<button>
<%= link_to 'See all Post', user_posts_path(@user) %>
</button>

File: app/views/users/index.html.erb [hunk 2]
```diff
@@ -1,72 +1,26 @@
...
-        <img src="https://www.fillmurray.com/640/360" class="img-fluid rounded-start" alt="...">
-      </div>
-      <div class="col-md-8">
-        <div class="card-body">
-          <h5 class="card-title">Card title</h5>
-          <p class="card-text">This is a wider card with supporting text below as a natural lead-in to additional content. This content is a little bit longer.</p>
-          <p class="card-text"><small class="text-muted">Last updated 3 mins ago</small></p>
-        </div>
-      </div>
+          <h5 class="card-title"> <%= userpost.title%></h5>
```
Comment (rloterh) [inline app/views/users/index.html.erb]: Your views are generating errors. Kindly ensure that the variable `userpost` is correctly spelt as indicated in the attached screenshot below:


image

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Atilla, 

You did extremely great on this project! 👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there! 🚦 🚦 🚦  

## Good Points 👍  
- Descriptive Pull Request
- No linter errors


## Required Changes ♻️
_Kindly check the comments under the review._


<br>
Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 1 more limited review per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/views/users/show.html.erb (hunk 1 above)
Comment (AtillaTahak) [reply app/views/users/show.html.erb]: Actually I changed my controller now come from posts controller view this view doesn't show and doesn't work . But you are right if I didn't use this view I should be deleted :/ thank you for review

**Review by HillaryOkello — APPROVED** [review]
# Hi @AtillaTahak,

## To highlight
- Descriptive pull request
- Professional README file

Nice work implementing the changes requested by the previous reviewer 👏🏻 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**